cwatch.php

```diff
@@ -200,29 +180,79 @@ function toggleLicenseField() {
         $package_type->attach($tooltip);
         $fields->setField($package_type);
 
-        // Set the license types
-        $license_types = Configure::get('cwatch.products');
-        foreach ($license_types as $license_type => $terms) {
-            $license_types[$license_type] = Language::_(
-                'CWatch.package_fields.license_' . strtolower($license_type),
-                true
-            );
+        // Set the module row to use if not given
+        if (empty($vars->module_row)) {
+            if (!empty($vars->module_group) && is_numeric($vars->module_group)) {
+                // Let the module decide which row to use
+                $vars->module_row = $this->selectModuleRow($vars->module_group);
+            } else {
+                $rows = (array) $this->ModuleManager->getRows($vars->module_id);
```

Comment:
Since this extends the Module class, you should be abl eto just call

```
$this->getModuleRows()
```

Reply:
Updated